src/api-reference/user-provisioning/spend/v4.spend-user-extension-schema.md

```diff
@@ -146,7 +146,7 @@ By default, the hierarchy structure is navigable via a tree control, and each no
 |`roleName`|`string`|-|**Required** Spend role for a Spend User.|
 |`roleGroups`|`string array`|-|**Required** Group(s) to be associated with the Spend role.|
 
-**NOTE:** Please refer to the [Spend Role Codes](./v4.spend-role-code-definition.html) page for the available value of `roleName`.
+**NOTE:** Please refer to the [Spend Role Codes](./v4.spend-role-code-definition.md) page for the available value of `roleName`.
```

Comment:
@wujasmine This needs to be .html for the page to display properly once published. Everything else looks fine.